main.go

```diff
@@ -77,3 +78,11 @@ func getEnvDefault(key string, defaultVal string) string {
 
 	return val
 }
+
+func getURLUsingPlek(key string) string {
+	url, err := plek.Find(key)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return url.String()
```

Comment:
@alext: can you help me understand how that would be beneficial? from our experience with plek.rb, do we think we need that?